OgreMain/include/Ogre.i

```diff
@@ -768,6 +769,10 @@ SHARED_PTR(Material);
 %ignore Ogre::Mesh::getBoneAssignmentIterator;
 %template(PoseList) std::vector<Ogre::Pose*>;
 %template(SubMeshList) std::vector<Ogre::SubMesh*>;
+%define %standard_byref_params(unsigned short)
```

Comment:
```suggestion
```

Reply:
We need to remove %enddef at the end of this block too I think